docs/metrics.md

```diff
@@ -43,64 +43,64 @@ Total VM filesystem capacity in bytes. Type: Gauge.
 Used VM filesystem capacity in bytes. Type: Gauge.
 
 ### kubevirt_vmi_memory_actual_balloon_bytes
-Current balloon bytes. Type: Gauge.
+Current balloon size in bytes. Type: Gauge.
 
 ### kubevirt_vmi_memory_available_bytes
-Amount of `usable` memory as seen by the domain. Type: Gauge.
+Amount of `usable` memory as seen by the domain. This value may not be accurate if a balloon driver is in use or if the guest OS does not initialize all assigned pages Type: Gauge.
 
 ### kubevirt_vmi_memory_domain_bytes_total
 The amount of memory in bytes allocated to the domain. The `memory` value in domain xml file. Type: Gauge.
 
 ### kubevirt_vmi_memory_pgmajfault
-The number of page faults when disk IO was required. Type: Counter.
+Page faults occur when a process makes a valid access to virtual memory that is not available. When servicing the page fault, if disk IO is required, it is considered as major fault. Type: Counter.
 
 ### kubevirt_vmi_memory_pgminfault
-The number of other page faults, when disk IO was not required. Type: Counter.
+Page faults occur when a process makes a valid access to virtual memory that is not available. When servicing the page fault, if disk IO is NOT required, it is considered as minor fault. Type: Counter.
 
 ### kubevirt_vmi_memory_resident_bytes
 Resident set size of the process running the domain. Type: Gauge.
 
 ### kubevirt_vmi_memory_swap_in_traffic_bytes_total
-Swap in memory traffic in bytes. Type: Gauge.
+The total amount of data read from swap space of the guest in Bytes. Type: Gauge.
 
 ### kubevirt_vmi_memory_swap_out_traffic_bytes_total
-Swap out memory traffic in bytes. Type: Gauge.
+The total amount of memory written out to swap space of the guest in Bytes. Type: Gauge.
 
 ### kubevirt_vmi_memory_unused_bytes
-Amount of `unused` memory as seen by the domain. Type: Gauge.
+The amount of memory left completely unused by the system. Memory that is available but used for reclaimable caches should NOT be reported as free. Type: Gauge.
 
 ### kubevirt_vmi_memory_usable_bytes
-The amount of memory which can be reclaimed by balloon without causing host swapping in bytes. Type: Gauge.
+How much the balloon can be increased in bytes without pushing the guest system to swap, corresponds to 'Available' in /proc/meminfo Type: Gauge.
```

Comment:
Technically correct, since the balloon is the pool of available memory, but confusing.
I liked the use of "reclaimed", which shows we're taking something away from the guest.

Reply:
Fixed.